src/forms/output/BenefitsLineGraph.js

```diff
@@ -48,8 +48,12 @@ class BenefitsLineGraph extends Component {
         interval  = Math.ceil((max / 100) / 10) * 10;
 
     var xRange      = _.range(limits.min, max, interval),  // x-axis/earned income numbers
-        extraProps  = { snap: { fill: false }, section8: { fill: false }},
+        extraProps  = {},
         datasets    = getChartData(xRange, multiplier, client, activePrograms, extraProps);
+    
+    for (let benefitIndex = 0; benefitIndex < activePrograms.length; benefitIndex++) {
+      extraProps[ activePrograms[ benefitIndex ] ] = { fill: false };
```

Comment:
This seems not to be working currently.

Note for future: May be worth abstracting some of the specific benefit chart frostings. @todo

Reply:
Ah, nevermind, I see it